### ISSUE TITLE
Add wrappers for group_session_builder & cipher

### ISSUE THREAD

**Issue by ma11hew28** [ChatSecure/SignalProtocol-ObjC#4]

**Comment by chrisballinger** [comment]
Are you working on this? Feel free to submit a pull request

**Comment by ma11hew28** [comment]
Yeah, we are. OK, we'll submit a pull request when we're done adding support for groups.

We're stuck on figuring out [how to get the serialized sender key distribution message](https://github.com/WhisperSystems/libsignal-protocol-c/issues/77). Do you have any suggestions?

**Comment by chrisballinger** [comment]
We actually don't use the sender key group stuff, and instead create a bunch of 1:1 sessions among all group participants (and all their devices).

**Comment by crossle** [comment]
@ma11hew28 pull request? if not done, i'm trying add group session

**Comment by ma11hew28** [comment]
@crossle we’re done. We’ll push the code soon. I’ll let you know.

**Comment by crossle** [comment]
@ma11hew28 Thanks, when push, i wanna try it

**Comment by crossle** [comment]
Any progress?
